src/main/resources/static/domain-events-api.yml

```diff
@@ -1334,6 +1370,48 @@ components:
         - reason
         - legacyReasonCode
         - destination
+    AssessmentAllocatedEnvelope:
+      type: object
+      properties:
+        id:
+          $ref: '#/components/schemas/EventId'
+        timestamp:
+          type: string
+          example: '2022-11-30T14:53:44'
+          format: date-time
+        eventType:
+          type: string
+          example: approved-premises.assessment.allocated
+        eventDetails:
+          $ref: '#/components/schemas/AssessmentAllocated'
+      required:
+        - id
+        - timestamp
+        - eventType
+        - eventDetails
+    AssessmentAllocated:
+      type: object
+      properties:
+        assessmentId:
+          $ref: '#/components/schemas/AssessmentId'
+        assessmentUrl:
+          $ref: '#/components/schemas/AssessmentUrl'
```

Comment:
It's good to include the ID and URL of the assessment being allocated, but I think we always need to include the ID and URL of the *application* also in our domain events. 

The application is the entity from which all related entities flow. For example when a Delius integration consumes a domain event it typically uses `applicationUrl` to provide a link back to the CAS1 service. 

See https://github.com/ministryofjustice/hmpps-probation-integration-services/blob/d59771747e6f2263e5f94509f7cbdba1f249993d/projects/approved-premises-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/approvedpremises/EventDetails.kt#L49-L71

Reply:
This has been done.